src/screens/AccountUnlock.js

```diff
@@ -56,8 +56,8 @@ export class AccountUnlockAndSign extends React.PureComponent {
 									}),
 									NavigationActions.navigate({ routeName: next })
 								],
-								index: 1, // FIXME workaround for now, use SwitchNavigator later: https://github.com/react-navigation/react-navigation/issues/1127#issuecomment-295841343
-								key: undefined
+								index: 1,
+								key: undefined // FIXME workaround for now, use SwitchNavigator later: https://github.com/react-navigation/react-navigation/issues/1127#issuecomment-295841343
```

Comment:
I previously missed this block, use `navigateToLegacyAccountList` from `navigationHelpers.js`